platform/packaging/build/pytorch/Dockerfile

```diff
@@ -0,0 +1,144 @@
+FROM debian:bookworm AS bookworm_cuda
+
+###
+#
+# Define environment
+
+WORKDIR /workspace
+
+ENV PYTORCH_VERSION="v2.0.1"
+ENV PATH="$PATH:/usr/local/cuda/bin"
+ENV DEBIAN_FRONTEND="noninteractive"
+ENV NVIDIA_DRIVER_CAPABILITIES compute,utility
+ENV NVIDIA_VISIBLE_DEVICES all
+###
+#
+# Workaround gcc-12 issue:
+# https://github.com/pytorch/pytorch/issues/77939#issuecomment-1526844015
+
+ENV CXXFLAGS='-Wno-maybe-uninitialized -Wno-uninitialized -Wno-free-nonheap-object'
+ENV CFLAGS='-Wno-maybe-uninitialized -Wno-uninitialized -Wno-free-nonheap-object'
+
+###
+#
+# Not all of these environment variables are used
+# This list of config options should be audited for correctness
+
+#ENV T_NCCL_INCLUDE_DIRS="PATH=/opt/nvidia/hpc_sdk/Linux_x86_64/23.7/comm_libs/12.2/nccl/include"
+#ENV T_NCCL_LIBRARIES="PATH=/opt/nvidia/hpc_sdk/Linux_x86_64/23.7/comm_libs/12.2/nccl/lib"
+#-DNCCL_INCLUDE_DIRS:${T_NCCL_INCLUDE_DIRS} -DNCCL_LIBRARIES:${T_NCCL_LIBRARIES} -DMAX_JOBS:${T_MAX_JOBS}
+ENV T_TORCH_CUDA_ARCH_LIST="STRING=8.0;8.6"
+ENV T_MAX_JOBS="INTEGER=30"
+ENV T_USE_CUDNN="BOOL=TRUE"
+ENV T_USE_CUSPARSELT="BOOL=TRUE"
+ENV T_USE_FBGEMM="BOOL=TRUE"
+ENV T_USE_KINETO="BOOL=TRUE"
+ENV T_USE_NUMPY="BOOL=TRUE"
+ENV T_USE_NNPACK="BOOL=TRUE"
+ENV T_USE_QNNPACK="BOOL=TRUE"
+ENV T_USE_SYSTEM_NCCL="BOOL=FALSE"
+ENV T_USE_NCCL="BOOL=FALSE"
+ENV T_BUILD_CAFFE2="BOOL=TRUE"
+ENV T_BUILD_CAFFE2_OPS="BOOL=TRUE"
+ENV T_USE_FLASH_ATTENTION="BOOL=TRUE"
+ENV T_BUILD_BINARY="BOOL=TRUE"
+ENV T_BLAS="BOOL=TRUE"
+ENV T_USE_ZSTD="BOOL=TRUE"
+
+
+#CUDA_TOOLKIT_ROOT_DIR="/usr/local/cuda"
+#CUDA_HOME="/usr/local/cuda"
+#CUDA_NVCC_EXECUTABLE="/usr/local/cuda/bin/nvcc"
+#ATEN_THREADING="NATIVE"
+#USE_SYSTEM_LIBS=1
+
+###
+#
+# Define dependencies
+
+RUN apt update
+RUN apt -y install build-essential
+RUN apt -y install ca-certificates
+#RUN apt -y install ccache
+RUN apt -y install cmake
+RUN apt -y install curl
+RUN apt -y install git
+RUN apt -y install python3
+RUN apt -y install libnuma-dev
+RUN apt -y install libssl-dev
+RUN apt -y install libzstd-dev
+
+###
+#
+# Not sure if or why these are needed
+
+RUN apt -y install libjpeg-dev
+RUN apt -y install libpng-dev
+
+COPY sources.list /etc/apt/
```

Comment:
`sources.list` isn't checked in, causing the build to fail for me.

Reply:
oh, right. sources.list is not needed. will update.